test-scripts/local-is-deployment/Solutions/solution01/src/test/jmeter/01-TestTomCat.jmx

```diff
@@ -0,0 +1,75 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This is a tomcat test script, and is not used in this PR. Shall we remove this for now?

Reply:
This will get executed from TG. Similar how tests are being executed under current implementation.